packages/arb-token-bridge-ui/src/components/TransferPanel/TokenRow.tsx

```diff
@@ -69,14 +73,23 @@ function BlockExplorerTokenLink({
   )
 }
 
-function TokenListInfo({ token }: { token: ERC20BridgeToken | null }) {
+function TokenListInfo({
+  token
+}: {
+  token: ERC20BridgeToken | NativeCurrencyEther | null
```

Comment:
Here, and in some of the following methods, we pass an additional `NativeCurrencyEther`. This way we can check for it and return ETH details as needed.